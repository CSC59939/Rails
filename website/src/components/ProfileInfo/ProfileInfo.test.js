import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Button, Card, Icon } from 'antd';
import { ProfileInfo } from '..';
import '../../utils/tests/test.css';

configure({ adapter: new Adapter() });

describe("ProfileInfo", () => {
  it('Render ProfileInfo without any error', () => { // eslint-disable-line no-undef
    expect(renderer.create(<ProfileInfo name='Student' email='Test@email.com'/>).toJSON()).toMatchSnapshot();
  });
})
