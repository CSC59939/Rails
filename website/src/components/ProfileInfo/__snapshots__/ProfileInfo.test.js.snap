// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfileInfo Render ProfileInfo without any error 1`] = `
<div
  className="ProfileInfo"
>
  <i
    className="ProfileImage material-icons"
    style={
      Object {
        "color": "black",
      }
    }
  >
    account_circle
  </i>
  <div
    className="rightSide"
  >
    <p
      className="name"
    >
      Student
    </p>
    <p
      className="email"
    >
      Test@email.com
    </p>
  </div>
</div>
`;
